core-bundle/src/Resources/contao/widgets/Password.php

```diff
@@ -140,12 +158,17 @@ protected function validator($varInput)
 	 */
 	public function generate()
 	{
+		// If there is a password, the field is no longer required
+		if ($this->varValue)
+		{
+			unset($this->arrAttributes['required']);
+		}
+
 		return sprintf(
-			'<input type="password" name="%s" id="ctrl_%s" class="tl_text tl_password%s" value="%s" autocomplete="new-password"%s onfocus="Backend.getScrollOffset()">%s%s',
+			'<input type="password" name="%s" id="ctrl_%s" class="tl_text tl_password%s" value="" autocomplete="new-password"%s onfocus="Backend.getScrollOffset()">%s%s',
 			$this->strName,
 			$this->strId,
 			($this->strClass ? ' ' . $this->strClass : ''),
-			($this->varValue ? '*****' : ''),
```

Comment:
Should we set a `placeholder="•••••"` if there is already a password set?
This would keep the feature that the user can see if the field is empty or not.

Reply:
Added in cddbc12a750e8f7a3ac0927ecce6641f39d69caa. I left the five stars, though, because `•••••` does not quite generate the expected output.

<img width="379" alt="" src="https://user-images.githubusercontent.com/1192057/191913200-3db724ab-377f-472f-a748-417fc1fe5f9a.png">

Maybe we should use a real placeholder that says "Set a new password" or something alike?

Reply:
<img width="346" alt="" src="https://user-images.githubusercontent.com/1192057/191913552-539f4f2c-994a-4e6f-b3f2-baa26403bc6a.png">

I guess the stars are fine, aren‘t they?